web/src/components/DRightClick/private/useContextMenu.ts

```diff
@@ -0,0 +1,44 @@
+import type { Ref } from 'vue'
+
+export default function (
+  containerRef: Ref<HTMLElement | null>,
+  menuRef: Ref<HTMLElement | null>
+) {
+  const showMenu = ref<boolean>(true)
+  const x = ref<number>(0)
+  const y = ref<number>(0)
+  const maxX = ref<number>(0)
+  const maxY = ref<number>(0)
```

Comment:
```suggestion
  const showMenu = ref(true)
  const x = ref(0)
  const y = ref(0)
  const maxX = ref(0)
  const maxY = ref(0)
```
这种不需要标注泛型，让 TS 自己推断就好了